src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.ts

```diff
@@ -195,7 +195,12 @@ export class ServiceFormComponent extends CdForm implements OnInit {
           })
         ]
       ],
-      group: [null, Validators.required],
+      group: [
+        null,
+        CdValidators.requiredIf({
+          service_type: 'nvmeof'
+        })
+      ],
```

Comment:
Thanks for fixing this.
This is breaking service form now for other components FYI
lets add this in description too

Reply:
yup, its there btw.

> additionally fixed a validation issue in the service form